lib/fib_source_b_impl.cc

```diff
@@ -31,340 +31,446 @@
 #include <boost/format.hpp>
 
 namespace gr {
-  namespace dab {
+    namespace dab {
 
 ////////////////////////////////////////
 /// MCI-FIGs, ready to transmit
 ////////////////////////////////////////
 
-    //ensemble info, CIF counter has to increase with each CIF
-    const char fib_source_b_impl::d_ensemble_info[56] = {0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0,
-                                                         0,
-                                                         0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-                                                         0,
-                                                         0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
-    // 000 00110 000 00000 0100000000000000 00 0 0000000000000 00000000
+        //ensemble info, CIF counter has to increase with each CIF
+        const char fib_source_b_impl::d_ensemble_info[56] = {0, 0, 0, 0, 0, 1,
```

Comment:
Do the line breaks have a "semantic" here? the array "formatting" doesn't seem to reflect the spaces in the binary string comment above

Reply:
This is caused to autoformat. I think we can cross the 80 column border in this case.